python/lsst/sims/catUtils/supernovae/snObject.py

```diff
@@ -589,6 +590,94 @@ def SNObjectSED(self, time, wavelen=None, bandpass=None,
         SEDfromSNcosmo.addCCMDust(a_x=ax, b_x=bx, ebv=self.ebvofMW)
         return SEDfromSNcosmo
 
+    def SNObjectSourceSED(self, time, wavelen=None):
```

Comment:
Should this be called `restFramSED()`?  Since it is a part of the class `SNObject`, I do not think it needs 'SNObject' in the method name, and 'restFrame' is what you are actually returning.